services/arrow-flight/src/main/java/com/dremio/service/flight/impl/FlightWorkManager.java

```diff
@@ -67,7 +68,8 @@ public FlightWorkManager(Provider<UserWorker> workerProvider,
    * @return A FlightPreparedStatement which consumes the result of the job.
    */
   public FlightPreparedStatement createPreparedStatement(FlightDescriptor flightDescriptor,
-                                                         Supplier<Boolean> isRequestCancelled, UserSession userSession) {
+                                                         Supplier<Boolean> isRequestCancelled,
```

Comment:
Are these formatting changes required? If not, can we avoid them as they clutter the review and make it harder to find what the actual changes were.

Reply:
Sure - we will avoid doing massive formatting